### PR TITLE
feat(post): ethereum blockchain getting started

### PR DESCRIPTION
**Pull request by asooge**

### CODE REVIEW

File: src/content/ethereum-blockchain-getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+## Start your Drizzle App
+
+1. Now that we have our Metamask and our contracts deployed to our local blockchain, lets go back to our frontend app, end the process (if still running) and restart:
+
+```bash
+npm start
+```
+
+1. We should now see our drizzle application that we can interact with our blockchain. Let's test the first contract, "Simple Storage" to make sure we can update, read, and persist data to our blockchain. Set the value to any number and submit. You should see the value update soon after.
```
Comment (trybick) [inline src/content/ethereum-blockchain-getting-started/index.md:172]: This one is showing as `1` on the page but I think should be `2`

File: src/content/ethereum-blockchain-getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+   ```bash
+   ganache-cli
+   ```
+
+   when successful, this command will provide you with 10 available accounts and corresponding private keys, each account with 100 ETH. Will also provide a mnemonic or 12-word phrase. The blockchain will listen on port 8545.
+
+1. Let's copy this 12-word mnemonic and create a command to run our blockchain as deterministic and mnemonic:
+
+   in our code editor, we are going to edit our package.json inside our truffle-example directory. We are going to create a command called `ganache` to run our blockchain and persist it to a db. In this example, I'm using the mnemonic: 'picture tray motion shuffle action fun puzzle velvet sample trip reject beef' but you can use the mnemonic that was generated by your ganache-cli or another mnemonic as you please. In order for our data to persist, we will need to run ganache-cli with the following flags:
```
Comment (trybick) [inline src/content/ethereum-blockchain-getting-started/index.md]: I think the formatting is a little off in this block. Might want to not have this one intended

**Review by trybick — COMMENTED** [review]
Looks great and like a lot of work!

File: src/content/ethereum-blockchain-getting-started/index.md (hunk 1 above)
Comment (asooge) [reply src/content/ethereum-blockchain-getting-started/index.md:172]: Thanks, good catch. Should be fixed with my latest commit.

File: src/content/ethereum-blockchain-getting-started/index.md (hunk 2 above)
Comment (asooge) [reply src/content/ethereum-blockchain-getting-started/index.md]: I think I'm okay with this being indented. This one is a long step but everything indented below is part of step 3.

**Review by trybick — APPROVED** [review]
I would check with fabio and veiko if they want to review too, but looks great

**Pull request merged**